stimela/tests/acceptance_tests/stimela-test-kat7.py

```diff
@@ -0,0 +1,461 @@
+                                "Output-Name": imname,
+                                "Image-NPix": 256,
+                                "Image-Cell": 30,
+                                "Cache-Reset": True,
+                                "Freq-NBand": 2,
+                                "Weight-ColName": "WEIGHT",
+                                "Beam-Model": "FITS",
+                                "Beam-FITSFile": "'JVLA-L-centred-$(xy)_$(reim).fits'",
+                                "Data-ChunkHours": 10,
+                                "Beam-FITSFeed" : "rl",
+                                "Data-Sort": True,
+                                "Log-Boring": True,
+                                "Deconv-MaxMajorIter": 1,
+                                "Deconv-MaxMinorIter": 500,
+                        },
+                        input=INPUT, output=OUTPUT, shared_memory="36gb",
+                        label="image_target_field_r0ddfacet:: Make a test image using ddfacet")
+
+
+                #Stitch LSMs together
+                lsm2=PREFIX+'-LSM2'
+#               recipe.add("cab/tigger_convert", "stitch_lsms1", {
+#                       "input-skymodel"  :   "%s.lsm.html:output" % lsm0,
+#                       "output-skymodel" :   "%s.lsm.html:output" % lsm1,
+#                       "rename"	  : True,
+#                       "force"           : True,
+#               },
+#                       input=INPUT, output=OUTPUT,
+#                       label="stitch_lsms1::Create master lsm file")
+
+                recipe.add('cab/casa_uvcontsub','uvcontsub',
+                        {
+                                "msname"         :    MS,
+                                "field"          :    "0",
+                                "fitorder"       :    1,
+                        },
+                        input=INPUT,
+                        output=OUTPUT,
+                        label='uvcontsub:: Subtract continuum in the UV plane')
+
+
+                #Image HI
+                recipe.add('cab/casa_clean', 'casa_dirty_cube',
+                        {
+                                "msname"         : MS+".contsub",
+                                "prefix"         : PREFIX,
+                                "mode"           : 'channel',
+                                "nchan"          : nchans,
+                                "niter"          : 0,
+                                "psfmode"        : 'hogbom',
+                                "threshold"      : '1mJy',
+                                "npix"           : 256,
+                                "cellsize"       : 30,
+                                "weight"         : 'natural',
+                                "port2fits"      : True,
+                        },
+                        input=INPUT,
+                        output=OUTPUT,
+                        label='casa_dirty_cube:: Make a dirty cube with CASA CLEAN')
+
```

Comment:
what happened to the lwimager step?

Reply:
Got a bit overzealous there, I'll put it back.